src/data/services/DiscoveryDataApiService.js

```diff
@@ -114,6 +114,21 @@ class DiscoveryDataApiService {
     return Promise.all(promises);
   }
 
+  static internalReviewEdit(courseRun) {
+    const queryParams = {
+      exclude_utm: 1,
+    };
+    const { key } = courseRun;
+    const body = Object.assign({}, courseRun);
+    const courseRunUrl = `${DiscoveryDataApiService.discoveryBaseUrl}/course_runs/${key}/`;
+    // Remove key param from body for API validation
```

Comment:
What's the necessity for this?

Reply:
when doing internal update, the api only accepts ofac_resctriction, ofac_comment, and status.  no need to pass key, it wont do anything with it.  will actually throw error for irrelevant params.